admissions/advfilters.py

```diff
@@ -0,0 +1,39 @@
+import django_filters
+
+from application.models import HighSchoolApplication
+
+FILTER_CHOICES = (("0", "Rejected"), ("1", "Accepted"), ("2", "Pending"))
```

Comment:
can we also show WithDrawn status on the admin side, and also hide Accept/reject buttons if the student withdraws his application

Reply:
Done